test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -402,9 +402,10 @@ def _obfuscate_acct_file(self, attrs_obf, file_path):
                 if not skip_record:
                     record = ";".join(record_list[:3]) + ";" + \
                         " ".join(["=".join(n) for n in kvl_list])
-                    fdout.write(record + "\n")
+                    newcontent.append(record + "\n")
 
-        shutil.move(fout, file_path)
+        with open(file_path, "w") as fd:
+            fd.write("".join(newcontent))
```

Comment:
Wouldn't it be better to use fd.writelines(newcontent)?

Reply:
I think writelines() is slower than doing a join and a write(), I saw posts like this and decided to just use write: https://stackoverflow.com/questions/44560655/python-writelines-and-write-huge-time-difference